components/index.ts

```
@@ -1,17 +1,23 @@
-  CustomTextInput as TextInput,
-  CustomSnackBar as SnackBar,
-  DropdownComponent as Select,
+  CustomBar as BarChart,
+  CustomChip as Chip,
   CustomImage as Image,
   CustomModal as Modal,
+  CustomSnackBar as SnackBar,
+  CustomTextInput as TextInput,
+  CustomeDatePicker as DatePicker,
+  DropdownComponent as Select,
   FullWidthImage as FullWidthImage,
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)